node/nodeRunner.go

```diff
@@ -778,7 +785,7 @@ func CreateManagedDataComponents(configs *config.Configs, managedCoreComponents
 		Core:                          managedCoreComponents,
 		EpochStartNotifier:            managedCoreComponents.EpochStartNotifierWithConfirm(),
 		CurrentEpoch:                  storerEpoch,
-		CreateTrieEpochRootHashStorer: configs.FlagsConfig.ImportDbSaveTrieEpochRootHash,
+		CreateTrieEpochRootHashStorer: configs.ImportDbConfig.ImportDbSaveTrieEpochRootHash,
```

Comment:
there is no check for importDBEnabled here. Should the SaveTrieEpochRootHash be enabled also when not running in importDb mode? If so, it should be move from the ImportDBConfig to a more general config location.

Reply:
was not decided if we need this besides the scope of the import-DB process.
Will keep it close to the import-db process, will make all flags from import-db config structure become false if the isInImportDBMode is false.

Reply:
done